Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -0,0 +1,78 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Doppler.HtmlEditorApi.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Doppler.HtmlEditorApi.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class CampaignsController : Controller
+    {
+        [HttpGet("/accounts/{accountName}/campaigns/{campaignId}/content/design")]
+        public string GetCampaign(string accountName, int campaignId)
+        {
+            var campaignJson = @"{
+                    ""counters"": {
+                        ""u_row"": 2,
+                        ""u_column"": 2,
+                        ""u_content_menu"": 1,
+                        ""u_content_heading"": 1
+                    },
+                    ""body"": {
+                        ""rows"": [],
+                        ""values"": {
+                            ""textColor"": ""#000000"",
+                            ""backgroundColor"": ""#e7e7e7"",
+                            ""backgroundImage"": {
+                                ""url"": """",
+                                ""fullWidth"": true,
+                                ""repeat"": false,
+                                ""center"": true,
+                                ""cover"": false
+                            },
+                            ""contentWidth"": ""500px"",
+                            ""contentAlign"": ""center"",
+                            ""fontFamily"": {
+                                ""label"": ""Arial"",
+                                ""value"": ""arial,helvetica,sans-serif""
+                            },
+                            ""preheaderText"": """",
+                            ""linkStyle"": {
+                                ""body"": true,
+                                ""linkColor"": ""#0000ee"",
+                                ""linkHoverColor"": ""#0000ee"",
+                                ""linkUnderline"": true,
+                                ""linkHoverUnderline"": true
+                            },
+                            ""_meta"": {
+                                ""htmlID"": ""u_body"",
+                                ""htmlClassNames"": ""u_body""
+                            }
+                        }
+                    },
+                    ""schemaVersion"": 6
+                }
+            ";
+            return campaignJson;
+        }
+
+        [HttpGet("/accounts/{accountName}/campaigns/{campaignId}/content/thumbnail")]
+        public string GetCampaignThumbnail(string accountName, int campaignId)
+        {
+            var uriCampaignThumbnail = "https://via.placeholder.com/200x200";
+            return uriCampaignThumbnail;
+        }
```

Comment:
What do you think about returning a redirect in place of a JSON? (I am not sure)

Reply:
Good catch 👍